koans/1.intro.js

```diff
@@ -7,7 +7,7 @@ import {
 } from 'tape'
 
 import {
-  arrow,
+  arr,
```

Comment:
So I've been thinking about this and I'm not entirely sure how to deal with the naming since Flow doesn't support declaring the type of something with the same name it will have in the actual code.

```js
type A = Number
const A = 1234
```

Yields an error, and it's workaround-able with:

```js
type _A = Number
const A: _A: 1234
```

Which is what I did for the `Left` and `Right` constructors for the `Either` type.

```js
type _Left<A> = ["Left", A]
const Left = (a): _Left<*> => ["Left", a]
// same for Right, with _Right

type Either<A, B> = _Left<A> | _Right<B>
```

so I'm wondering, since the convention in Haskell for lifting a function into an arrow is `arr`, should we continue down this path?

Should we provide an `_Arrow` type instead with an `Arrow` constructor? After considering `arr` itself is not a monadic constructor for a given type, but a category lifter (since Arrow is not a kind of monad but a [Category itself](https://hackage.haskell.org/package/base-4.9.1.0/docs/src/Control.Arrow.html#Arrow)) I'd argue that we shouldn't provide a standard constructor.

It's still up to debate whether we should call it `arr` or `arrow`, but that's minor since it can be renamed on importing (or both can be exported).